src/ecrc-api/pom.xml

```diff
@@ -68,35 +67,21 @@
 			<artifactId>spring-boot-starter-security</artifactId>
 		</dependency>
 		<dependency>
-			<groupId>javax.xml.bind</groupId>
```

Comment:
Migrate deprecated javax.xml.bind packages to jakarta.xml.bind to resolve java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException